src/plugins/vis_type_timeseries/public/application/components/vis_types/gauge/vis.js

```diff
@@ -10,9 +10,10 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { visWithSplits } from '../../vis_with_splits';
 import { createTickFormatter } from '../../lib/tick_formatter';
-import _, { get, isUndefined, assign, includes } from 'lodash';
+import { get, isUndefined, assign, includes, gt, gte, lt, lte, isEqual } from 'lodash';
 import { Gauge } from '../../../visualizations/views/gauge';
-import { getLastValue } from '../../../../../common/get_last_value';
+import { getLastValueOrEmpty } from '../../../../../common/last_value_utils';
+const OPERATORS = { gt, gte, lt, lte, eq: isEqual };
```

Comment:
what do you think about using `isNull/isEmpty` instead of isEqual. I think it that case from `getLastValue` method we can return null instead of empty array

Reply:
I dislike, that it will change the way of handling values. Right now all functions are taking 2 parameters. If we will add isNull/isEmpty, it will pass normally, but that is not good if we know, that some function will ignore our arguments.

But I can do it, no problem, as it will not cause any harm)